pkg/cidata/schema.go

```diff
@@ -0,0 +1,40 @@
+package cidata
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"strings"
+
+	"github.com/santhosh-tekuri/jsonschema/v5"
+	"gopkg.in/yaml.v3"
+)
+
+const schemaURL = "https://raw.githubusercontent.com/canonical/cloud-init/main/cloudinit/config/schemas/schema-cloud-config-v1.json"
```

Comment:
What's the license of this file?

Reply:
According to https://github.com/canonical/cloud-init/blob/main/LICENSE the repo is under your choice of GPL-3 or Apache-2.

Reply:
Will add a license statement for the file.